sunpy/net/dataretriever/tests/test_goes_ud.py

```diff
@@ -129,3 +130,17 @@ def test_time_for_url(LCClient, time):
     times = LCClient._get_time_for_url(url)
 
     assert all([tr == t2 for t2 in times])
+
+
+
+@pytest.mark.remote_data
+@pytest.mark.parametrize("time,instrument", [
+    (a.Time('2019/05/13 00:00', '2019/05/13 00:10'), a.Instrument('suvi')),
+])
+def test_query(time, instrument):
```

Comment:
This test should be in `test_goes_suvi.py` and you should use the `suvi_client` fixture in that file.

Reply:
done.